src/table/virtualizedDataTable.jsx

```diff
@@ -1460,101 +1641,161 @@ class VirtualizedDataTable extends Component {
       }
     }
 
-    const keysToSniff = [
-      'ArrowLeft',
-      'ArrowRight',
-      'ArrowUp',
-      'ArrowDown',
-      'Backquote',
-      'Backslash',
-      'Backspace',
-      'BracketLeft',
-      'BracketRight',
-      'Comma',
-      'Delete',
-      'Enter',
-      'Equal',
-      'Escape',
-      'Minus',
-      'Period',
-      'Quote',
-      'Semicolon',
-      'Slash',
-      'Space',
-      'Tab',
-    ];
-    for (let keyOn = 0; keyOn < 10; keyOn += 1) {
-      keysToSniff.push(`Digit${keyOn}`);
-    }
-    for (let keyOn = 48; keyOn < 91; keyOn += 1) {
-      keysToSniff.push(`Key${String.fromCharCode(keyOn)}`);
```

Comment:
Determine if we need to apply `scrollToColumn` to snap a column into view (like when walking with arrow keys from frozen to non-frozen which was scrolled such that the leftmost column needs to be scrolled into view